pkg/manager/task_export.go

```diff
@@ -135,6 +138,17 @@ func (t *ExportTask) ExportScenes(ctx context.Context) {
 			newSceneJSON.Markers = append(newSceneJSON.Markers, sceneMarkerJSON)
 		}
 
+		for _, movie := range movies {
+			if movie.Name.Valid {
+				sceneMovieJSON := jsonschema.SceneMovie{
+					MovieID:        movie.ID,
```

Comment:
JSON data identifies other objects either by some handle (such as name), or by checksum. ID should not be here.